### ISSUE TITLE
Cannot read property 'loginOrRegister' of undefined in 2.9.9 

### ISSUE THREAD

**Issue by adicatur** [qiscus/qiscus-sdk-web-core#56]
After upgrade the version from 2.9.7 to 2.9.9 get this error:

<img src="https://d1edrlpyc25xu0.cloudfront.net/sdksample/image/upload/IGgmxOQ292/Screen+Shot+2020-06-08+at+23.07.23.png" alt="setUser()" width="360"/>

but when set timeout the issue disappears

**Comment by adicatur** [comment]
or when do this, I don't see this issue
```javascirpt
qiscus.init( ... ).then(function () {
    qiscus.setUser()
 });

**Comment by ramzitm91** [comment]
any solution? I had the same issue :(

**Comment by adicatur** [comment]
hi @ramzitm91, you can update the latest one, 2.10.6

**Comment by ramzitm91** [comment]
thank for the response @adicatur but I already update it to the latest 2.10.7 but the issue is still there :( 
![image](https://user-images.githubusercontent.com/19144081/102849702-47ee4b80-444a-11eb-84e5-c6e5f944e13c.png)

**Comment by adicatur** [comment]
hi @ramzitm91 that one has solved in 2.10.8 but still on test will let you know once it good to go, you can downgrade to 2.10.6
